servicecontrol/index.md

```diff
@@ -6,3 +6,36 @@ isLearningPath: true
 ---
 
 include: servicecontrol
+
+How ServiceControl, ServicePulse and ServiceInsight work together can be found with [the Particular Service Platform](/platform/).
+
+### ServiceControl instance types
+
+After [installation](/servicecontrol/installation.md), the ServiceControl Management Utility provides the ability to add, upgrade and delete ServiceControl instances. There are three different instances that can be created:
+
+- [Error instances](/servicecontrol/servicecontrol-instances/)  
+  This is the most commonly used ServiceControl instance and indispensable for the correct working of any NServiceBus system. Together with SerivcePulse it provides the ability to show and retry failed messages.
```

Comment:
Isn't it so that you can have up to one error instance but multiple audit instances? Doesn't it make sense to mention that here?

Reply:
```suggestion
- [Error instances](/servicecontrol/servicecontrol-instances/)  
  This is the most commonly used ServiceControl instance and indispensable to ensure the correct working of an NServiceBus system. Together with SerivcePulse, it provides the ability to visualize and retry failed messages.
```